src/renderer/qt/qt_window_manager.cc

```diff
@@ -341,16 +341,61 @@ void FillCandidates(const commands::Candidates &candidates,
 }
 }  // namespace
 
+struct VirtualRect {
+  const QScreen *screen;
+  const Rect rect;
```

Comment:
I think this member should be non-const.

Reply:
Context: https://abseil.io/tips/177

Reply:
`VirtualRect` is now a class and removed `const`.